server/api/sectionevent/sectionevent.controller.js

```diff
@@ -98,8 +103,11 @@ exports.index = function(req, res) {
 
     query.sort({time: -1});
 
-    query.lean().execAsync()
+    query.execAsync()
       .then((sectionEventsArray)=>{
+        sectionEventsArray = sectionEventsArray.map((sectionEvent)=>{
+          return sectionEvent.toObject();
+        })
         var events = {};
```

Comment:
```javascript
sectionEventsArray = sectionEventsArray.map((sectionEvent)=> sectionEvent.toObject());
```
shortening the syntax helps with cleanliness IMO the brackets and return aren't necessary